docs/book/orders/checkout.rst

```diff
@@ -252,6 +252,23 @@ and apply a proper transition and flush the order via the manager.
 
 The Checkout is finished after that.
 
+Checkout related events
+-----------------------
+
+On each step of checkout a dedicated event is triggered.
+
++-----------------------------------------+
+| Event id                                |
++=========================================+
+| ``sylius.address``                      |
```

Comment:
Actually it will dispatch two events `sylius.pre_address` and `sylius.post_address`. Take a look at
https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ResourceBundle/Controller/EventDispatcher.php#L21
and
https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ShopBundle/Resources/config/routing/checkout.yml#L16